.github/workflows/build-test-package.yml

```diff
@@ -3,22 +3,22 @@ name: Build, test, package
 on: [push,pull_request]
 
 env:
-  itk-git-tag: "171fb2ba33a87041f99328a2f26612ff33aa9cc8"
-  itk-wheel-tag: "v5.3rc04.post3"
+  itk-git-tag: "abf5fa10522a36bc51f42f20f426a622f42ed90d"
+  itk-wheel-tag: "v5.3rc04.post4"
 
 jobs:
   build-test-cxx:
     runs-on: ${{ matrix.os }}
     strategy:
       max-parallel: 3
       matrix:
-        os: [ubuntu-20.04, windows-2019, macos-11]
+        os: [ubuntu-20.04, windows-2022, macos-11]
```

Comment:
Should we bump this to `ubuntu-22.04`?

Reply:
OK, if CI comes green, merge.

Reply:
Added https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/issues/10 to track ubuntu upgrade